main.go

```diff
@@ -22,27 +22,30 @@ func getListenAddress() string {
 	return fmt.Sprintf("%v:%d", *address, *port)
 }
 
+var l *log.Logger
```

Comment:
Why is this a global variable? Couldn't it be made part of the struct?

Reply:
Fixed as well